### ISSUE TITLE
Avoiding duplication of properties between spring boot and gradle

### ISSUE THREAD

**Issue by Dzeri96** [liquibase/liquibase-gradle-plugin#124]
I'm not experienced with gradle so forgive me if this is very obvious, but I'm trying to have the plugin use the same configuration as it would when running in spring boot context.

For example, one would have the following properties files:
```
application.properties <- Properties that are the same across all environments
application-prod.properties <- Some properties overriden here
application-dev.properties <- Some properties overriden here
``` 
Let's also say that some of my properties look like this `spring.datasource.password=${DATASOURCE_PASSWORD:test_pw}`

How can I avoid duplicating all of these properties between gradle and spring? Also, what would be the best way to chose the spring env when running the plugin? 

It's essentially the same question as [this one on Sackoverflow](https://stackoverflow.com/questions/68159509/how-to-parse-spring-format-application-properties-file-outside-spring-applicatio), but it has no answers.

**Comment by landsman** [comment]
Yeah, the current approach is overcomplicated. It should be loaded from properties as a plugin for Maven in an ideal world.

**Comment by stevesaliman** [comment]
I'm not sure there is a good way to do this.  Spring applications shouldn't know anything about how they were built, and Gradle shouldn't know anything about how Spring works.  The only thing I can think of is to create a build task that creates a "application-liquibase.properties" file in the same directory as "application.properties".  Liquibase.properties can contain whatever build properties you want Spring to know about, and Spring can then just include the "liquibase" profile.
